.github/labeler.yml

```
@@ -40,16 +54,28 @@
 
 "Info / User Interface":
   - "**/ui/**/*"
+  - "**/ui**/*"
   - "**/widget/**/*"
   - "src/list_view.*"
   - "src/surroundings_menu.*"
 
+"Items: Armor / Clothing":
+  - "**/armor/**/*"
+
+"Items: Containers":
+  - "**/containers/**/*"
```

```suggestion
  - "**/**containers**"
```
There aren't any container folders, just files, btw this would also catch changes to containers folders
```
find -type f | grep 'containers'
./data/mods/innawood/recipes/tools_containers.json
./data/mods/Magiclysm/items/containers.json
./data/mods/DinoMod/recipes/containers.json
./data/mods/DinoMod/items/containers.json
./data/mods/Aftershock/items/containers.json
./data/json/recipes/tools/containers.json
./data/json/items/containers.json
```